packages/react-next/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -404,14 +490,8 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
 
     event.persist();
     // Call onChange after state is updated
-    this.setState(
-      {
-        selectedIndices: newIndexes,
-      },
-      () => {
-        this._onChange(event, options, index, checked, multiSelect);
-      },
-    );
+    this.props.hoisted.setSelectedIndices(newIndexes);
+    this._onChange(event, options, index, checked, multiSelect);
```

Comment:
Calling `onChange` immediately after `setSelectedIndices` (instead of in the post-`setState` callback) is a subtle behavior change, but I'm not sure how to tell if it will matter for consumers.

Reply:
I believe that this is okay. onChange doesn't really rely on anything that would happen from the newIndexes so it should be alright.